menu.js

```diff
@@ -0,0 +1,17 @@
+const menuBars = document.querySelector('.menu-bars');
+const mobMenu = document.querySelector('#mob-menu');
+const mobCloseMenu = document.querySelector('#mob-close-menu');
+
+const mobileNav = document.querySelector('#mob-menu>ul>li>a');
+
+menuBars.addEventListener('click', () => {
+  mobMenu.style.display = 'flex';
+});
+
+mobCloseMenu.addEventListener('click', () => {
+  mobMenu.style.display = 'none';
+});
```

Comment:
- [ ] The mobile menu should disappear when the user clicks(or taps) any option of the mobile menu.
In your code there are a little few issues, as you can see here:

_Please try to avoid this issue! 😊_

![review](https://user-images.githubusercontent.com/69011963/142645685-24813f9b-546b-44c2-b8de-a60fcfdfdd1c.gif)
: